### PR TITLE
Added new unreleased directory in order to capture the release notes during development

### PR DESCRIPTION
**Pull request by daravep**

### CODE REVIEW

File: unreleased/TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Release of Daml DAML_VERSION
+
+Canton CANTON_VERSION has been released on RELEASE_DATE. You can download the Daml Open Source edition from the Daml Connect [Github Release Section](https://github.com/digital-asset/daml/releases/tag/vCANTON_VERSION). The Enterprise edition is available on [Artifactory](https://digitalasset.jfrog.io/artifactory/canton-enterprise/canton-enterprise-CANTON_VERSION.zip).
```
Comment (mziolekda) [inline unreleased/TEMPLATE.md:3]: Shouldn't this text be talking about Daml rather than canton?

**Comment by garyverhaegen-da** [comment]
It's unclear to me how this is intended to integrate with the overall release process on this repo. Could you flesh out the interactions in [the release process documentation](https://github.com/digital-asset/daml/blob/main/release/RELEASE.md) as part of this PR?

**Review by miklos-da — COMMENTED** [review]
Aren't we already tracking these items in a separate Google Doc?

**Comment by garyverhaegen-da** [comment]
> Aren't we already tracking these items in a separate Google Doc?

Not that I'm aware; do you have a link to the spreadsheet you're thinking of?

**Review by mziolekda — APPROVED** [review]

**Pull request merged**